run.py

```diff
@@ -10,52 +9,27 @@
 import sqlite3
 import sys
 import time
+import psycopg2
 
 from balloon import *
 from telemetry import *
 
+def to_list(tuple):
+    return list(tuple)
+
```

Comment:
Why is this necessary? Isn't it possible to `map(list, sqldata)` without this wrapper function?

Reply:
Its way more clear and understandable, using this 2 line function. Because for those who don't know how does map works, this 2 line is easy to understand that it basically converts tuples to lists.

Reply:
I struggle to understand how

```py
def to_list(tuple):
    return list(tuple)

tabledata = map(to_list, sqldata)
````

is more clear than
```py
tabledata = map(list, sqldata)
```

it's just a tiny abstraction and you still need to understand how map works. Don't think this is worth discussing, both approaches work fine :)

Reply:
First code that you write is not a usage of functions in python
Normal usage is:
`def to_list(tuple):
    return list(tuple)

tabledata = to_list(sqldata)`

And in that way you don't need to use map(). But as you mentioned, yeah both approaches work fine but this one is easier to understand for python developers.

Reply:
That's exactly how `to_list` is used in this PR
https://github.com/sm3ulc/hab-wspr/pull/7/files#diff-d6af0459a37d985953d7040c14f53feb3b9cc9e58b543aa3c2b80256d276c5e0R30

Reply:
Oh, noticed that just now. Probably map is taking just the function name, didn't need to use it before, my bad mate.

Reply:
In Javascript and functional programming, it is common to see something like this
```js
[1, 2, 3].map(toString); // toString is a function
// which basically translates to
[toString(1), toString(2), toString(3)]
```
Cheers!